html_reference.html

```diff
@@ -0,0 +1,80 @@
+					</div>
+				</div>
+			</div>
+			<div class="aboutMePosition">
+				<h2 class="container">employment_history  <i class="fas fa-briefcase" id="briefcase"></i></h2>
+				<div class="aboutMeContent">
+					<div class="aboutMeText">
+						Palmyra is an ancient city in present-day Homs Governorate, Syria. Inhabited since the Neolithic period, it entered recorded history in the early second millennium 						BC. Palmyra became part of the Roman Empire in the first century AD. Palmyrene merchants established colonies along the Silk Road and the city grew wealthy from 						trade caravans. Many monumental projects were erected, such as the Great Colonnade, the Temple of Bel, and distinctive tower tombs. Palmyra reached the apex of its 						power in the 260s, when its king Odaenathus defeated the Persian emperor Shapur I.
+					</div>
+				</div>
+			</div>
+			<div class="aboutMePosition">
+				<h2 class="container">extra_curricular <i class="fas fa-biking" id="biking"></i></h2>
+				<div class="aboutMeContent">
+					<div class="aboutMeText">
+						Palmyra is an ancient city in present-day Homs Governorate, Syria. Inhabited since the Neolithic period, it entered recorded history in the early second millennium 						BC. Palmyra became part of the Roman Empire in the first century AD. Palmyrene merchants established colonies along the Silk Road and the city grew wealthy from 						trade caravans. Many monumental projects were erected, such as the Great Colonnade, the Temple of Bel, and distinctive tower tombs. Palmyra reached the apex of its 						power in the 260s, when its king Odaenathus defeated the Persian emperor Shapur I.
+					</div>
+				</div>
+			</div>
+		</div>
+		<h3 class="container">portfolio  <i class="fas fa-palette" id="palette"></i></h3>
+			<div class="porfolioButtonsContainer">
+				<!-- id's used for button background images -->
+				<a href="#" class="portfolioButtons" id="button1">PROJECT1</a>
+				<a href="#" class="portfolioButtons" id="button2">PROJECT2</a>
+				<a href="#" class="portfolioButtons" id="button3">PROJECT3</a>
+				<a href="#" class="portfolioButtons" id="button4">PROJECT4</a>
+				<br>
+				<a href="#" class="portfolioButtons" id="button5">PROJECT5</a>
+				<a href="#" class="portfolioButtons" id="button6">PROJECT6</a>
+				<a href="#" class="portfolioButtons" id="button7">PROJECT7</a>
+				<a href="#" class="portfolioButtons" id="button8">PROJECT8</a>
+			</div>
+		<footer>
+			<!-- inconsistent class naming is used due to fab being links to specific logos from an external website -->
+			<div class="socialMediaButtons">
+				<a><i class="fab fa-linkedin"></i></a>
+				<a><i class="fab fa-github-square"></i></a>
+				<a><i class="fas fa-share-alt-square"></i></a>
+			</div>
+		</footer>
+		<div class="textAdjust">
+			<div class="anchorBar"></div>
+			<a href="#" class="pressHereAnchor"> << <i class="fas fa-gamepad" id="gamepad"></i> >> </a>
+			<div class="anchorBar"></div>
```

Comment:
The two empty divs can be removed and replaced with top/bottom margin applied to the textAdjust class

Reply:
done